project/ISSUES.md

```diff
@@ -0,0 +1,179 @@
+Galaxy Issue Management
+=======================
+
+The purpose of this document is to formalize how we manage the full cycle
+(reporting, tracking progress, resolution) of both feature/enhancement ideas
+and bugs, as well as providing a few general guidelines for longer term
+planning for various Galaxy related projects.  Significant inspiration taken
+from the way the [Docker](https://github.com/docker/docker) project manages
+issues.
+
+We've tried several different approaches over the course of the project and two
+common problems we've tried to address in this approach are:
+
+* The issue graveyard -- where, once off the first 'page' of issues or below
+  the 'fold' in Trello, things sometimes do not resurface and are lost to
+  history.
+
+* Difficulty clearly presenting and maintaining a high level project roadmap
+  and associated meta-issues.
+
+
+Milestones
+==========
+
+Every pull request should, prior to merge, be assigned to the milestone
+corresponding to the Galaxy release it will first appear in (15.10, 16.01, and
+so on).  This, along with the tags applied, will be used to generate the
+release notes.
+
+Any non-PR issue assigned to a milestone *must* be resolved or reassigned prior
+to publishing that release -- that is, this is the new way to create and track
+"Issues Blocking Release".  This will be a primary way to report bugs and force
+them to be reconciled (fixed, closed, or intentionally postponed) prior to
+release.  In practice, bugs should almost always be tagged with a milestone
+which forces the reconciliation date.  Issues *may* be, but they don't
+necessarily have to be -- this is subjective and it depends on if the submitter
+(or any other contributor) wants to force the issue to be revisited at the next
+release.
+
+Effective use of milestones should prevent bugs from falling through the
+cracks, and will provide a mechanism for forcing the revisitation(and thus
+progress or even potential discard) of ideas for enhancements or features.
+
+
+Labeling Structure
+==================
+
+To allow for easy search, filtering, and general issue management every issue
+or PR (not tagged `Procedures` or `Planning`) is *required* to have three
+labels which indicate the type, status, and focus area of the issue.  Any issue
+without these three tags will be automatically have a `triage` label applied
+indicating that it needs human intervention to be correctly tagged.  These
+`triage` tagged issues will be regularly reviewed and tagged as appropriate.
+
+Kind Labels
+-----------
+
+The 'kind' label set is used for classifying the type of contribution or
+request/report to separate enhancements and new features from bugs, etc.
+
+* `kind/bug` - something is broken, and it needs fixing
+* `kind/documentation` - documentation is unclear or can be improved
```

Comment:
If it is in this repository, it is both code and documentation. For the docs - The READMEs are markup rendered by Github, the RST is compiled by readthedocs. For the code - "programs must be written for people to read, and only incidentally for machines to execute" [SICP](https://mitpress.mit.edu/sicp/front/node3.html) - on the first page of the best book about coding ever written.

Stepping down from the ivory tower though, if you think documentation for in repository might be a good place to approach the project from, I would say it should be an ``area`` tag then, since those are the tags I would use to find something useful to work on if I were on the outside - that and beginner friendly.